pkg/providers/aws/cluster_vpc.go

```diff
@@ -459,8 +460,32 @@ func getSubnets(ec2Svc ec2iface.EC2API) ([]*ec2.Subnet, error) {
 	return subs, nil
 }
 
+func getRHMIVpc(ctx context.Context, c client.Client, ec2Svc ec2iface.EC2API, logger *logrus.Entry, clusterID, organizationTag string) (*ec2.Vpc, error) {
```

Comment:
What's the reasoning for having this in `cluster_vpc`?

I think `getStandaloneVpc` might be more generic

Reply:
The reason it's here is I was using the getClusterVpc logic as it's very similar. Can definitely move it somewhere more appropriate. @ciaranRoche also mentioned to not use rhmi specifically so will update the method name to the more generic one that you've suggested there (getStandaloneVpc) 👍